DotnetFoundation/DotnetFoundation.Api/wwwroot/Templates/Emails/BaseTemplate.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
```

Comment:
You don't need wwwroot folder. We can remove it.

Reply:
This is created by Shreyansh , i am following his structure only